src/comp_math.cc

```diff
@@ -12,32 +12,29 @@ namespace compmath {
 
 CompMap Add(const CompMap& v1, const CompMap& v2) {
   CompMap out(v1);
-  CompMap vv2(v2);
   for (CompMap::const_iterator it = v2.begin(); it != v2.end(); ++it) {
     int nuc = it->first;
-    out[nuc] += vv2[nuc];
```

Comment:
since this is an optimization PR, is there any reason to skip the `nuc` definition and just have `out[it->first] += it->second;`?